### ISSUE TITLE
Notification: Payment Receipt to Customer

### ISSUE THREAD

**Issue by wtogami** [slickage/adness#25]
After any successful payment a receipt should be sent to the customer for their own records containing pertinent details of the invoice, the BTC amount and USD value at the time of the transaction.

**Comment by taesup** [comment]
I've implemented this but have left out the BTC amount and USD value in the body of the email. This information is found in the invoice itself, right? I did add more information about any further actions they need to accomplish (create an ad for the registration email, checking their ads for auction wins)

**Comment by wtogami** [comment]
The reason full details are needed in the e-mail sent at the time of purchase is customers need the record to be available much later, perhaps a year or more after the event.  Pretty much everything you purchase online e-mails the full receipt in this manner for this reason as it least cuts reliance on the need for the server to retain those details.

I agree implementing it as such is not critical in the short-term.  I'd like to help polish up this aspect of the customer experience later.

**Comment by taesup** [comment]
Agreed. I'll move this on to a later phase mostly because I don't have that information saved on the server right now.
